README.md

```diff
@@ -166,14 +165,14 @@ If you don't have xclip or xsel, micro will use an internal clipboard for copy a
 
 ### Colors and syntax highlighting
 
-If you open micro and it doesn't seem like syntax highlighting is working, this is probably because
+If you open micro and find out that syntax highlighting is working, this is probably
 you are using a terminal which does not support 256 color. Try changing the colorscheme to `simple`
-by pressing CtrlE in micro and typing `set colorscheme simple`.
+by pressing `Ctrl + E` in micro and typing `set colorscheme simple`.
 
-If you are using the default Ubuntu terminal, to enable 256 make sure your `TERM` variable is set
+If you are using the default Ubuntu terminal and wish to enable 256, you need to make sure your `TERM` variable is set
```

Comment:
~~If you are using the default Ubuntu terminal and wish to enable 256, you need to make sure your…~~ → If you are using the default Ubuntu terminal and wish to enable 256-color support, make sure your…